Documentation/BuildInstructionsLadybird.md

```diff
@@ -82,6 +82,11 @@ sudo apt install qt6-multimedia-dev
 sudo pacman -S --needed autoconf-archive automake base-devel ccache cmake curl libgl nasm ninja qt6-base qt6-multimedia qt6-tools qt6-wayland ttf-liberation tar unzip zip
 ```
 
+Or you can install through AUR :
+```
+yay -S ladybird
```

Comment:
I don't think we we should imply a specific AUR helper. IMO it would make more sense to have the generic AUR instructions (i.e. `git clone <url>`  and `makepgk -sri` or similar) and then add something along the lines of "or use your favorite AUR helper".